client/src/components/categorical/category/index.js

```diff
@@ -144,15 +144,14 @@ class Category extends React.PureComponent {
     */
     const { schema } = annoMatrix;
     const { colorAccessor, colorMode } = colors;
-    const { genesets, differential } = this.props;
```

Comment:
if you delete the use of `differential`, please also delete the setting of the property (line 48, above)